Include/internal/pycore_regions.h

```diff
@@ -57,6 +57,7 @@ bool _Pyrona_AddReferences(PyObject* target, int new_refc, ...);
 #endif
 
 int _Py_CheckRegionInvariant(PyThreadState *tstate);
+void _PyErr_Region(PyObject *tgt, PyObject *new_ref, const char *msg);
```

Comment:
Does this need to be in the header?  I don't think it will be used outside regions.c

Reply:
Ok, I didn't know if we wanted to raise a region error elsewhere. I'll make it static for now

Reply:
I would have thought these errors would come from WriteBarriers?